moped-editor/src/layouts/DashboardLayout/NavBar/DropdownMenu.js

```diff
@@ -162,20 +162,20 @@ const DropdownMenu = ({
         {helpItems.map((item) => {
           if (item.linkType === "external") {
             return (
-              <MenuItem key={item.link} onClick={handleDropdownClose}>
```

Comment:
it looks like you need to move the `key` prop up to the `Link` component. i'm getting the warning:

```
DropdownMenu.js:116 Warning: Each child in a list should have a unique "key" prop.
```